pcdsdevices/interface.py

```diff
@@ -487,6 +490,13 @@ def signal_info(signal):
                 units=units)
 
 
+def base_info(positioner):
```

Comment:
```suggestion
def positionerbase_info(positioner):
    """Return basic status information for the given positioner."""
```

Reply:
good call, I originally wrote this for ophydobj but changed my mind